stable/provazio/templates/dashboard-deployment.yaml

```diff
@@ -0,0 +1,139 @@
+{{- if .Values.dashboard.enable }}
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "provazio.dashboard.name" . }}
+  labels:
+    app: {{ template "provazio.dashboard.name" . }}
+    chart: {{ template "provazio.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.dashboard.replicas }}
+  selector:
+    matchLabels:
+      provazio/service: {{ template "provazio.dashboard.name" . }}
+  template:
+    metadata:
+      labels:
+        provazio/service: {{ template "provazio.dashboard.name" . }}
+    spec:
+      containers:
+      - env:
+        - name: PROVAZIO_ENV
```

Comment:
Shouldn't it be in a `ConfigMap` or `Secret`? Too many `env`

Reply:
Secret stuff are in a secret. Don't want to change to ConfigMap right now, can do in the future if we choose